python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
+    MAX_DISPLACEMENT = 10**7
+
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        self._info_script = {
+            "file": create_label(""),
+            "progress": self._create_progress_bar(0, 100, 0),
+        }
+
+        self._labels_force = {
+            "axial_min": create_label(""),
+            "axial_max": create_label(""),
+            "axial_total": create_label(""),
+            "tangent_min": create_label(""),
+            "tangent_max": create_label(""),
+            "tangent_total": create_label(""),
+        }
+
+        self._buttons_script = {
```

Comment:
Your only usage of hash map is to call it again with keys you know in _create_group_script_control. This can be much better solved by creating a function taking array, which will put passed widgets into grid. So you will call (see cRIOpy..ValueGrid):
```
layout.addWidget(ButtonGrid([set_button("Load Script", self._callback_script_load_script), set_button("Run", self._callback_script_command,,,,, 2)
```

where two is for number of rows, and ButtonGrid will work like that:
```
class ButtonGrid(QWidget):
    def __init__(self, buttons, rows=2):
        super().__init__()
        cols = len(buttons) / rows
        layout = QGridLayout()
        r = c = 0
        for b in buttons:
            layout.addWidget(b, r, c)
            c += 1
            if c > cols:
                r += 1
                c = 0

        self.setLayout(layout)
```

with of course some function also an option.